gallery/src/styles/NxCode/NxCodePage.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxCodeExample from './NxCodeExample';
+
+const nxCodeExampleCode = require('!!raw-loader!./NxCodeExample').default;
+
+const NxCodePage = () => {
+  return (
+    <>
+      <GalleryDescriptionTile>
+        <p className="nx-p">
+          Using the &lt;code&gt; HTML tag with the <code className="nx-code">.nx-code</code> className applies a
+          monospace font and other styling to make your code snippets stand out.
```

Comment:
`.nx-code` doesn't actually apply a monospace font.  User-agent styles for `<code>` do that.

Reply:
> `.nx-code` doesn't actually apply a monospace font. User-agent styles for `<code>` do that.

Now `nx-code does it: https://github.com/sonatype/sonatype-react-shared-components/pull/69/commits/a1c5d4c9375d633896b0a21584c35816865a8388